### ISSUE TITLE
Option for even slower playback

### ISSUE THREAD

**Issue by pudgerockstar0001** [incidentist/the_tuul#38]
There are times when even the 0.7 playback isn't slow enough to help synch really fast lyrics.
This especially becomes a challenge in very fast rap parts.
One of the ways I would try to work around it was to just skip syllables and word cuts and just treat entire line as one word being sung and press space at the start of the line and again at the next line.
However, even this fails sometimes. A slower playback like 0.5 or 0.25 might be able to help in these cases.

**Comment by incidentist** [comment]
Makes sense. This should be simple. Things are a bit busy now, but I will try to add it before the end of June.
